code/modules/mob/living/simple_mob/subtypes/mechanical/disbot_vr.dm

```diff
@@ -0,0 +1,50 @@
+/mob/living/simple_mob/mechanical/infectionbot
+	name = "Strange robot"
+	desc = "You get the feeling you should run."
+	icon = 'icons/mob/vagrant_vr.dmi'
+	icon_state = "vagrant"
+	icon_living = "vagrant"
+	icon_dead = "vagrant"
+	icon_gib = "vagrant"
+
+	maxHealth = 65
+	health = 40
+	movement_cooldown = 4
+
+	response_help = "pets the"
+	response_disarm = "gently pushes aside the"
+	response_harm = "hits the"
+	faction = "vagrant"
+	harm_intent_damage = 3
+	melee_damage_lower = 6
+	melee_damage_upper = 9
+	light_color = "#8a0707"
+	attacktext = "mauled"

```

Comment:
Robot is supposed to be trying to attack via wild injection stabs, no? Why 'mauled ' verb?

Reply:
Oversight, I'll fix it.